src/commands/transaction/cosign.ts

```diff
@@ -71,6 +72,10 @@ export default class extends ProfileCommand {
             .pipe(
                 switchMap((addresses) => sequentialFetcher.getResults(addresses)),
                 flatMap((transaction: AggregateTransaction[]) => transaction),
+                tap((transaction) => {
```

Comment:
Apparently, the table is displayed for every aggregate transaction to cosign, not just the transaction that it is being cosigned.